pkg/machines/libvirt.es6

```diff
@@ -796,12 +828,18 @@ function handleEvent(dispatch, connectionName, line) {
 }
 
 function startEventMonitor(dispatch, connectionName) {
-    var output_buf = '';
+    let output_buf = '';
 
+    dispatch(checkLibvirtStatus());
     // set up event monitor for that connection; force PTY as otherwise the buffering
     // will not show every line immediately
     cockpit.spawn(['virsh'].concat(VMS_CONFIG.Virsh.connections[connectionName].params).concat(['-r', 'event', '--all', '--loop']), {'err': 'message', 'pty': true})
         .stream(data => {
+            if (data.startsWith("error: Disconnected from") || data.startsWith("error: internal error: client socket is closed")) {
+                // libvirt failed
```

Comment:
can we log here? Just to be sure we do not swallow status info which might be useful when debugging

Reply:
added debug log to all the places where event monitor fail is expected